server.lua

```diff
@@ -429,17 +429,62 @@ end)
 RegisterNetEvent('qb-vehicleshop:server:checkFinance', function()
     local src = source
     local player = QBCore.Functions.GetPlayer(src)
+    local cash = player.PlayerData.money['cash']
```

Comment:
These vars should be moved down into the auto finance loop code block. Otherwise, they won't update whenever a payment is made.